code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -36,6 +36,9 @@
 	var/weapon_orig_force = 0
 	var/chosen_name
 
+	/// The time it takes for the cleanbot to clean something.
+	#define CLEANING_TIME (1 SECONDS)
```

Comment:
Do one of two things:

1. Change this to just a variable
2. Move this define to before the Initialize proc itself, then undef afterwards.